NVDAVersions.json

```diff
@@ -0,0 +1,32 @@
+[
+	{
+		"NVDAVersion": "2019.3",
+		"apiVer": "2019.3",
+		"backCompatTo": "2019.3"
```

Comment:
I don't doubt that you have looked in NVDA to determine this is the earliest API version, but could you add to the readme some info about this?

Also worth pointing out in an "discussion / explanation" section a benefit to sorting files into folders by API version is that this covers all releases prior to `2019.3`. If the folders were named by NVDA version, a folder would be needed for every prior version even though they had no unique API version number (or any concept of it)

Older versions of NVDA (than the release where this lands) won't have the add-on store built in, but will still be able to manually browse any websites that expose this data and download older versions. For that there is value in allowing them in the store.